ipaserver/install/httpinstance.py

```diff
@@ -172,8 +173,8 @@ def remove_httpd_ccaches(self):
         # Make sure that empty env is passed to avoid passing KRB5CCNAME from
         # current env
         ipautil.remove_file(paths.HTTP_CCACHE)
-        for f in os.listdir(paths.IPA_CCACHES):
-            os.remove(os.path.join(paths.IPA_CCACHES, f))
+        shutil.rmtree(paths.IPA_CCACHES)
+        ipautil.run([paths.SYSTEMD_TMPFILES, '--create'])
```

Comment:
You could use systemd-tmpfiles prefix option to only create the directory and any subdirectory. The option is documented as

> Only apply rules with paths that start with the specified prefix.

I verified that ``[paths.SYSTEMD_TMPFILES, '--create', '--prefix', paths.IPA_CCACHES]`` creates a missing paths.IPA_CCACHES directory.

Reply:
Nice catch, I'll add that, thanks.